styles/google/catppuccin.user.css

```diff
@@ -1216,7 +1217,17 @@
     color: @subtext0;
   }
   body {
-    #catppuccin(@flavor);
+    #catppuccin(@lightFlavor);
+  }
```

Comment:
I don't think we need this anymore? We just need the media queries below so.

Reply:
yeah probs